datasets/multi_eurlex/README.md

```diff
@@ -0,0 +1,309 @@
+---
+pretty_name: MultiEURLEX
+
+annotations_creators:
+- found
+
+language_creators:
+- found
+
+languages:
+- en
+- da 
+- de 
+- nl 
+- sv 
+- bg 
+- cs 
+- hr 
+- pl 
+- sk 
+- sl 
+- es 
+- fr 
+- it 
+- pt 
+- ro 
+- et 
+- fi 
+- hu 
+- lt 
+- lv 
+- el 
+- mt
+
+licenses:
+- cc-by-sa-4.0
+
+multilinguality:
+- multilingual
+
+size_categories:
+- 10K<n<100K
+
+source_datasets:
+- original
+
+task_categories:
+- text-classification
+
+task_ids:
+- multi-label-classification
```

Comment:
We can add the task "topic-classification"

+ nit: formatting - remove empty lines 
```suggestion
pretty_name: MultiEURLEX
annotations_creators:
- found
language_creators:
- found
languages:
- en
- da 
- de 
- nl 
- sv 
- bg 
- cs 
- hr 
- pl 
- sk 
- sl 
- es 
- fr 
- it 
- pt 
- ro 
- et 
- fi 
- hu 
- lt 
- lv 
- el 
- mt
licenses:
- cc-by-sa-4.0
multilinguality:
- multilingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- text-classification
task_ids:
- multi-label-classification
- topic-classification
```

Reply:
Sure, I updated the README, accordingly.